recipes/muarch/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "muarch" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e81649e7eef494d9a10e20db6f87cc4edf45b19076ec5334bfd250881cb8636f
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
Please use:

```
script: "{{ PYTHON }} -m pip install . -vv"
```

and add `pip` as a `host` dependency.

Reply:
Adopted recommendations